index.js

```diff
@@ -43,10 +43,13 @@ function createWindow() {
         width: 640,
         height: 480
     });
-    window.loadFile('index.html');
+    window.loadFile('public/index.html');
     window.on('closed', () => {
         window = null;
     });
+
+    window.openDevTools();
```

Comment:
This should be available through the menu bar, or through a keyboard shortcut. No reason to open it every single time the application launches.

Reply:
I forgot to remove that line, I use it while developing. Now it is removed.